### DISCUSSION TITLE
"wired" transport of IR commands ?

### DISCUSSION THREAD

**Discussion by Onepamopa** [Arduino-IRremote/Arduino-IRremote#1054] [General]
Hey guys,

Wondering if someone has tried this - using an ESP+digital optocoupler, in essence - removing the IR receiver on the AC unit and connecting the optocoupler instead. I think it should work, both the IR receiver and the optocoupler work using the same principle - the ESP would control the LED side, the AC should receive the command on the other end of the optocoupler.

H11L1 or VOH1016AD should (imho) do the job.

Any thoughts? Anyone tried something like this?

**Comment by Franck78** [comment]
A remote is an IR emitter , a varying air gap, an IR receiver
Making it with a fixed air gap won't change anything as long as the device (optocoupler) can switch as least at 38Khz.

I think the filter (as in a TSOP18438) is unnecessary because there is no interference light.

**Comment by ArminJo** [comment]
@Onepamopa What do you think is the reason of the `USE_NO_SEND_PWM` macro/compile switch?
